environments/docker-compose.common.yml

```diff
@@ -38,23 +32,26 @@ services:
       - JB_ADDITIONAL_ALLOWED_HOSTS=*
 
       - APP_ID=1ec756a5-e04a-4885-87ad-3325bbdd4872
-      - AWS_SECRET_ACCESS_KEY
+
+      # Variables which are passed through to allow users to set them
       - AWS_ACCESS_KEY_ID
-      - AWS_SESSION_TOKEN
+      - AWS_SECRET_ACCESS_KEY
       - AWS_SECURITY_TOKEN
+      - AWS_SESSION_TOKEN
       - JB_GCLOUD_PRIVKEY
       - JB_GITHUB_FETCHAPP_CREDS
-      - JB_REDSHIFT_CONNECTION
+      - JB_HIPAA
```

Comment:
This is the line of code that actually solves OPS-1828.

Reply:
me and casey weren't sure if JB_HIPAA is correct, as the environment variable that is getting pulled in is called `HIPAA_MODE` not sure if that is what is going on here? either way I don't feel 100% confident to be the person reviewing this

Reply:
There are two separate things: the name of a python variable defined in django settings, and the name of an environment variable.

See this line:

https://github.com/juiceinc/fruition/blob/develop/fruition/settings/base.py#L397

The environment variable is named `JB_HIPAA`. The convention is that all of the environment variables that JB consumes are prefixed with `JB_`. But the name of the variables in the django settings module can be anything, and don't need a `JB_` prefix because that would be redundant.